Userland/DevTools/HackStudio/LanguageServers/Cpp/CppComprehensionEngine.cpp

```diff
@@ -516,6 +516,12 @@ void CppComprehensionEngine::update_declared_symbols(DocumentData& document)
     set_declarations_of_document(document.filename(), move(declarations));
 }
 
+void CppComprehensionEngine::update_todo_entries(DocumentData& document)
+{
+    auto todo_entries = document.parser().get_todo_entries();
```

Comment:
nitpick: no need for the `todo_entries` variable here

Reply:
Done :D